xapian-letor/configure.ac

```diff
@@ -34,9 +34,14 @@ dnl 0:0:0 1.3.1 First release with xapian-letor split out from xapian-core
 LIBRARY_VERSION_INFO=0:0:0
 AC_SUBST(LIBRARY_VERSION_INFO)
 
+dnl Where xapian-letor.h, etc go.  In development release append "/xapian-1.3".
+dnl incdir=$includedir
+incdir=$includedir/xapian-1.3
+AC_SUBST([incdir])
```

Comment:
The commit message talks about an install path, but this is really just about the include directory so it'd be good to adjust the message to match.